Mathlib/Algebra/Polynomial/Div.lean

```diff
@@ -251,6 +264,84 @@ theorem divByMonic_eq_zero_iff [Nontrivial R] (hq : Monic q) : p /ₘ q = 0 ↔
     have : ¬degree q ≤ degree p := not_le_of_gt h
     unfold divByMonic divModByMonicAux; dsimp; rw [dif_pos hq, if_neg (mt And.left this)]⟩
 
+section mem_pow_natDegree_mul
+
+local notation "°" => Polynomial.natDegree
+local notation3 "coeff("p")" => Set.range (coeff p)
+local notation3 "submodule("p")" => 1 ⊔ Submodule.span ℤ coeff(p)
+
+attribute [local instance 9999] MulLeftMono.toPosMulMono MulRightMono.toMulPosMono
```

Comment:
It seems worth explaining why this is useful

Reply:
Turns out I can do without